### DISCUSSION TITLE
During field creation, disable editing should differentiate between creating and editing

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13647] [Feature Requests]
During field setup, you're asked if this field should be disabled during editing, but when creating a new record, you're not really editing, but creating. Maybe this disabling feature could be reworked into a 'disable during create' and 'disable during edit' and then perhaps rules could be automatically added / removed for the field to disable it 

e.g. if id is null then disable and vice versa

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
